quiz.html

```
@@ -0,0 +1,182 @@
+
+<!-- ANSWERS -->
+        <div
+  class="answers-container"
+  id="answers-container">
+</div>
+
+
+        <!-- ANSWERS -->
+        <div
+          class="answers-container"
+          id="answers-container">
```

HTMLHint (1.9.2)

[error] The id value [ answers-container ] must be unique.

(id-unique)